ui-core/src/components/MultipleCombobox.tsx

```diff
@@ -21,11 +28,12 @@ type MultipleComboboxProps<ComboboxItem> = {
  * many more options than a normal React-select would, while remaining performant.
  */
 export const MultipleComboBox = <ComboboxItem, >(props: MultipleComboboxProps<ComboboxItem>) => {
```

Comment:
Not a part of the original PR scope, but this might be a good time to make the MultipleComboBox have the same styling as the rest of the survey js components, as it has a different background, arrow, font, etc. from the other survey questions

![image](https://github.com/broadinstitute/juniper/assets/17440010/1aa59884-0b33-4479-8fbf-34275c781aa4)

Reply:
It has bugged me a little bit as well :) I took a quick pass at this but I couldn't get it looking close enough to the built-in SurveyJS components, so I'm going to merge as-is